frontend/src/lib/k8s/resourceQuota.ts

```diff
@@ -42,35 +42,114 @@ class ResourceQuota extends makeKubeObject<KubeResourceQuota>('resourceQuota') {
 
   get requests(): string[] {
     const req: string[] = [];
-    Object.keys(this.spec.hard).forEach(key => {
-      if (key.startsWith('requests.')) {
-        req.push(`${key}: ${this.status.used[key]}/${this.spec.hard[key]}`);
-      }
-    });
+    this.spec.hard &&
+      Object.keys(this.spec.hard).forEach(key => {
+        if (key === 'cpu' || key === 'memory' || key.startsWith('requests.')) {
+          req.push(
+            `${key}: ${this.normalizeUnit(key, this.status.used[key])}/${this.normalizeUnit(
+              key,
+              this.spec.hard[key]
+            )}`
+          );
+        }
+      });
     return req;
   }
 
   get limits(): string[] {
     const limits: string[] = [];
-    Object.keys(this.spec.hard).forEach(key => {
-      if (key.startsWith('limits.')) {
-        limits.push(`${key}: ${this.status.used[key]}/${this.spec.hard[key]}`);
-      }
-    });
+    this.spec.hard &&
+      Object.keys(this.spec.hard).forEach(key => {
+        if (key.startsWith('limits.')) {
+          limits.push(
+            `${key}: ${this.normalizeUnit(key, this.status.used[key])}/${this.normalizeUnit(
+              key,
+              this.spec.hard[key]
+            )}`
+          );
+        }
+      });
     return limits;
   }
 
   get resourceStats() {
     const stats: { name: string; hard: string; used: string }[] = [];
-    Object.keys(this.status.hard).forEach(key => {
-      stats.push({
-        name: key,
-        hard: this.status.hard[key],
-        used: this.status.used[key],
+    this.status.hard &&
+      Object.keys(this.status.hard).forEach(key => {
+        stats.push({
+          name: key,
+          hard: `${this.normalizeUnit(key, this.status.hard[key])} (${this.status.hard[key]})`,
+          used: `${this.normalizeUnit(key, this.status.used[key])} (${this.status.used[key]})`,
```

Comment:
If the result within parenthesis is the same as outside, then we should not include the parenthesis, otherwise it's `0 (0)` and it's a bit confusing.
See this example when using this PR (also shows `5 (5)`):
![Screenshot with the example](https://user-images.githubusercontent.com/1029635/235873576-94f4e6ed-8ff5-48ab-b837-6b9543835ee9.png)

Reply:
Instead of showing both normalized and unnormalized values I changed it to just show normalized values and the unnormalized value is shown as tooltip.

Reply:
To clarify. I think it's fine, and even probably recommended, to see the values right away. I just don't want all the values to be repeated. `21.27 GB (20 Gi)` is fine. `5 cores (5 cores)` or `0 (0)` is what I wanted to avoid. In this case we should just show `5 cores`.

I have tested your latest changes and the tooltip on hover effect is not clear (no indication it'll have that info unless the user actually hovers) so you might as well keep both values like before but special case when the values are the same, as mentioned.
Also, I see that in some cases the number + unit are separated by a space, in others they are not (and we should pluralize `core`):
![Screenshot showing the changes](https://github.com/headlamp-k8s/headlamp/assets/1029635/515511e8-6e36-498d-bbb0-34f58cc2c8ca)

Reply:
Fixed it.
<img width="620" alt="image" src="https://github.com/headlamp-k8s/headlamp/assets/22731013/6828462e-60eb-419f-ae88-2b74e48816ab">
<img width="575" alt="image" src="https://github.com/headlamp-k8s/headlamp/assets/22731013/f0a856ba-f172-4374-9206-2f1d98e698ad">